bin/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Check if .env file already exists
+if [ -f .env ]; then
+  echo ".env file already exists. Please remove it first."
+  exit 1
+fi
+
+# Check if .env.example file exists
+if [ ! -f .env.example ]; then
+  echo ".env.example file does not exist. Please create one."
+  exit 1
+fi
+
+# Copy .env.example to .env
+cp .env.example .env
+
+# Read the values of the variables from command-line arguments
```

Comment:
Shouldn't we sanitize these inputs, and make sure they're part of the set we have defined in the `.env.example` file? Technically a user could run the `install` command with any param

Reply:
I'm not sure what sanitisation would do for us here? I'm not sure there's a security risk.... the user could throw their laptop out of their window too if they wanted 😄

Reply:
It does a find and replace on values in the sample file though, so if they're not in there they won't be replaced.

Reply:
Yeah that latter is what I meant, not actual input sanitisation

Reply:
Surely this is a good thing. The sample file becomes our API, and if we want to add a new env var (or remove one) we only need to change the sample and it just works?

Reply:
Unless there's something I'm missing. Maybe you could provide an example of problematic usage?